src/Character.java

```diff
@@ -0,0 +1,115 @@
+
+public abstract class Character {
+
+	// Attributes
+	private static int count = 0;
+	private int id;
+	private byte level;
+	private static byte maxLevel = 50;
+	private static short baseAtack = 5;
+	private static short baseDefense = 2;
+	private static short baseHp = 50;
+	private String name;
+	
+	// Constructor
+	public Character(byte level, String name) {
+		this.setId();
+		this.setLevel(level);
+		this.setName(name);
+	}
+
+	// Methods
+	/**
+	 * Abstract method to calculate the attack (damage) of a character
+	 * 
+	 * This method will be defined and implemented by the child classes
+	 * */
+	public short calculateAttack() {
+		short random = (short)(Math.random() * (1 - 0)); 
+		return (short) (Character.baseAtack  + ( Character.baseAtack * random ) * Character.baseHp);
```

Comment:
Here it should be this.level instead of Character.baseHp

Reply:
Done!